### PR TITLE
binlog: use sql manage binlog

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@CaitinChen @ericsyh PTAL

File: dev/reference/tools/tidb-binlog/operation.md [hunk 1]
```diff
@@ -130,3 +130,63 @@ Usage of binlogctl:
     ```
 
     该命令会生成一个文件 `{data-dir}/savepoint`，该文件中保存了 Drainer 初次启动需要的 tso 信息。
+
+## 使用 TiDB SQL 管理 Pump/Drainer
+
+目前支持通过在 TiDB 中执行 SQL 来查看/管理 binlog 相关的状态。
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/operation.md]:
```suggestion
目前支持通过在 TiDB 中执行 SQL 语句来查看和管理 binlog 相关的状态。
```

File: dev/reference/tools/tidb-binlog/operation.md [hunk 2]
```diff
@@ -130,3 +130,63 @@ Usage of binlogctl:
...
+
+- 查看 TiDB 是否开启 binlog
+
+```bash
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/operation.md]: Please indent this code block with four spaces.

File: dev/reference/tools/tidb-binlog/operation.md [hunk 3]
```diff
@@ -130,3 +130,63 @@ Usage of binlogctl:
...
+| log_bin       |  ON   |
++---------------+-------+
+
+```
+
+值为 `ON` 时表示 TiDB 开启了 binlog。
+
+- 查看 Pump/Drainer 状态
+
+```bash
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/operation.md]: ditto

File: dev/reference/tools/tidb-binlog/operation.md [hunk 4]
```diff
@@ -130,3 +130,63 @@ Usage of binlogctl:
...
++--------|----------------|--------|--------------------|---------------------|
+| NodeID |     Address    | State  |   Max_Commit_Ts    |    Update_Time      |
++--------|----------------|--------|--------------------|---------------------|
+| pump1  | 127.0.0.1:8250 | Online | 408553768673342237 | 2019-05-01 00:00:01 |
++--------|----------------|--------|--------------------|---------------------|
+| pump2  | 127.0.0.2:8250 | Online | 408553768673342335 | 2019-05-01 00:00:02 |
++--------|----------------|--------|--------------------|---------------------|
+```
+
+```bash
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/operation.md]: ditto

File: dev/reference/tools/tidb-binlog/operation.md [hunk 5]
```diff
@@ -130,3 +130,63 @@ Usage of binlogctl:
...
+| drainer1 | 127.0.0.3:8249 | Online | 408553768673342532 | 2019-05-01 00:00:03 |
++----------|----------------|--------|--------------------|---------------------|
+| drainer2 | 127.0.0.4:8249 | Online | 408553768673345531 | 2019-05-01 00:00:04 |
++----------|----------------|--------|--------------------|---------------------|
+
+```
+
+- 修改 Pump/Drainer 状态
+
+```bach
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/operation.md]: ditto

File: dev/reference/tools/tidb-binlog/operation.md [hunk 6]
```diff
@@ -130,3 +130,63 @@ Usage of binlogctl:
...
+mysql> change pump to node_state ='paused' for node_id 'pump1'";
+Query OK, 0 rows affected (0.01 sec)
+```
+
+```bach
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/operation.md]: ditto

**Comment by WangXiangUSTC** [comment]
@lilin90 PTAL again

**Comment by lilin90** [comment]
@ericsyh PTAL

**Comment by ericsyh** [comment]
LGTM

**Pull request merged**